packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -1073,48 +1204,23 @@ class FlameChartNode<T extends FlameChartDataMixin<T>> {
     // TODO(kenz): this is intended to improve performance but can probably be
     // improved. Perhaps we should still show a solid line and fade it out?
     if (zoomedWidth < 0.5) {
-      return SizedBox(width: zoomedWidth);
+      return;
     }
 
     selected = selectable ? selected : false;
     hovered = selectable ? hovered : false;
-
-    final node = Container(
-      key: hovered ? null : key,
-      width: zoomedWidth,
-      height: rect.height,
-      padding: const EdgeInsets.symmetric(horizontal: 6.0),
-      alignment: Alignment.centerLeft,
-      color: _backgroundColor(
+    final drawnRect = Rect.fromLTWH(topLeft.dx, topLeft.dy, rect.width * zoom, rect.height * zoom);
```

Comment:
no need to scale height by zoom. zoom only affects horizontal size of the flame chart and nodes